integr8ly/integreatly-operator
# MGDAPI-10370 Update RHSSO operator to v12.0.1

# Description
Bump RHSSO operator version to v12.0.1 and operand to 7.4-16

## Verification
1. Install RHOAM/RHMI from this branch and verify operand and operator versions match the above version number. 
2. Ensure RHSSO has started correctly as there is a change to the authdelay jar in this PR and it must be added to the classpath. 
3. Verify version 1.0.2 of the authdelay.jar has been downloaded in the extensions container of the keycloak pod

## Type of change
